project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -11,6 +11,22 @@
  * Add a cancel button to all node forms.
  */
 function c4m_content_form_node_form_alter(&$form, &$form_state, $form_id) {
+  // Checks if the form's content type is being used as a message,
+  // to add a send notification option.
+  if (in_array($form_state['node']->type, c4m_message_valid_note_types())) {
+    // Checks if the form is for a published content.
+    $published = $form_state['node']->vid ? TRUE : FALSE;
```

Comment:
`is_new` doesn't exist on `$form_state['node']` - changing to `nid` and moving statement to the form actions array